sdk/src/loom/ModestMaps/core/painter/TileQueue.ls

```diff
@@ -39,19 +39,25 @@ package loom.modestmaps.core.painter
         
         public function sortTiles(callback:Function):void
         {
-            queue = queue.sort(callback) as Vector.<Tile>;
+            queue.sort(callback) as Vector.<Tile>;
```

Comment:
Yeah it is in place. But the cast does nothing since you don't return/store the value...